openlibrary/templates/site/neck.html

```diff
@@ -6,7 +6,9 @@
         <meta name="robots" content="$ctx.robots" />
 
     $if ctx.get("description"):
-    	<meta name="description" content="$ctx.description" />
+        <meta name="description" content="$ctx.description" />
+    $else:
+        <meta name="description" content="Open Library is an open, editable library catalog, building towards a web page for every book ever published." />
```

Comment:
This needs to be internationalized if you want it to make any sense in social shares.

Reply:
Okay thanks :)